<?xml version="1.0"?>

<launch>
	<!-- visualizer for object detection results publishes data for RViz visualization -->
	<node name="object_detection_visualizer" pkg="cob_object_detection_visualizer" type="object_detection_visualizer" output="screen">
	<!--rosparam command="load" file="$(find cob_object_detection_visualizer)/ros/launch/object_detection_visualizer_params.yaml"/-->
	<!--remap from="detection_array_topic" to="/surface_classification/object_detections"/-->
	<remap from="detection_array_topic" to="/label_reading/read_label/text_tag_detections"/>
	<remap from="pointcloud" to="/camera/depth_registered/points"/>
	<remap from="pointcloud_info" to="/camera/depth_registered/camera_info"/>
	
	<param name="display_rviz_markers" value="true" />
	<param name="display_detection_image" value="true" />
</node>

</launch>
